spec/controllers/posts_controller_spec.rb

```diff
@@ -25,11 +25,6 @@
         expect(response).to be_success
       end
 
-      it 'succeeds on mobile' do
-        get :show, "id" => @message.id
-        expect(response).to be_success
-      end
-
       it 'succeeds after removing a mention when closing the mentioned user\'s account' do
         user = FactoryGirl.create(:user, :username => "user")
         alice.share_with(user.person, alice.aspects.first)
```

Comment:
Might be worth to see if some tests can be extracted to specifically testing the service and replaced with call assertions here or even completely dropped.